schemas/content/product-attribute.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-attribute",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Attribute",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Generic product attribute of a product in the product catalog.",
+  "definitions": {
+    "attribute": {
+      "properties": {
+        "xdm:key": {
+          "title": "Key",
+          "type": "string",
+          "description": "The technical key / identifier of this attribute."
+        },
+        "schema:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "The human-readable name of the attribute."
+        },
+        "xdm:value": {
+          "title": "Value",
+          "description": "The attribute value."
+        },
+        "xdm:unit": {
```

Comment:
Should be an enumeration of common units.

Reply:
See: https://github.com/adobe/xdm/issues/334

We need to get a decision on how we want to implement open enums.

Reply:
Ok I think we need to wait for #334 then.

Reply:
`xdm:unit` is fine for now, I'd standardize the list of known units here: #358

Reply:
I reworked the entire attribute value to be either a `simple` string or `https://ns.adobe.com/xdm/data/measure` be be more standardized and relay on things which are already there.